### ISSUE TITLE
1.4-Pixel Perfect Layout-Tyler A. Reckart

### ISSUE THREAD

**Issue by tylerreckart** [TIY-GVL-FEE-2015-August/assignments#64]
### Tasks

* **Getting Started**
    * [x] Follow the assignment instructions to create a new issue.
    * [x] Follow the assignment instructions to create a new repository.
    * [x] Create and commit `index.html`
    * [x] Follow the assignment instructions to create a new pull request.
    * [x] **Read [this article](http://diveintohtml5.info/semantics.html) about
      semantic HTML.**
* **Make It So**
    * [x] Checkpoint: Basic HTML structure committed!
    * [x] Checkpoint: Basic layout CSS committed!
    * [x] Checkpoint: Pixel perfect page header committed!
    * [x] Checkpoint: Pixel perfect page footer committed!
    * [x] Checkpoint: Pixel perfect column layout committed!
    * [x] Checkpoint: Pixel perfect layout and typography committed!
    * [x] Checkpoint: Links with hover states committed!
    * [x] Mission complete!

**Comment by tylerreckart** [comment]
Viewable at : http://tylerreckart.github.io/1.4-Pixel-Perfect-Layout/
Repo: https://github.com/tylerreckart/1.4-Pixel-Perfect-Layout/

Learning zone: 3

Time worked: ~25 hours

Results: Finished. I am happy with the outcome, however I wish I had more time to work on the responsive version. I was able to get everything as close as I could within reason. The image section at the bottom is slightly different from the original outline. I decided to turn it into a moving carousel to showcase even more possible posts. The project was not difficult, it was more time consuming than anything. This morning I was able to completely reorganize and optimize my code, which cleaned it up quite a bit and gave it a much quicker load time.
